examples/go/go-client/helper/rest/api/v2010/model_test_response_object.go

```diff
@@ -22,36 +22,36 @@ import (
 
 // TestResponseObject struct for TestResponseObject
 type TestResponseObject struct {
-	AccountSid                 *string                                 `json:"account_sid,omitempty"`
-	Sid                        *string                                 `json:"sid,omitempty"`
-	TestString                 *string                                 `json:"test_string,omitempty"`
-	TestInteger                *int                                    `json:"test_integer,omitempty"`
-	TestObject                 *TestResponseObjectTestObject           `json:"test_object,omitempty"`
-	TestDateTime               *string                                 `json:"test_date_time,omitempty"`
-	TestNumber                 *float32                                `json:"test_number,omitempty"`
-	PriceUnit                  *string                                 `json:"price_unit,omitempty"`
-	TestNumberFloat            *float32                                `json:"test_number_float,omitempty"`
-	TestEnum                   *string                                 `json:"test_enum,omitempty"`
-	TestArrayOfIntegers        []int                                   `json:"test_array_of_integers,omitempty"`
-	TestArrayOfArrayOfIntegers [][]int                                 `json:"test_array_of_array_of_integers,omitempty"`
-	TestArrayOfObjects         *[]TestResponseObjectTestArrayOfObjects `json:"test_array_of_objects,omitempty"`
+	AccountSid                 *string                                      `json:"account_sid,omitempty"`
+	Sid                        *string                                      `json:"sid,omitempty"`
+	TestString                 *string                                      `json:"test_string,omitempty"`
+	TestInteger                *int                                         `json:"test_integer,omitempty"`
+	TestObject                 *TestResponseObjectTestObject                `json:"test_object,omitempty"`
+	TestDateTime               *string                                      `json:"test_date_time,omitempty"`
+	TestNumber                 *float32                                     `json:"test_number,omitempty"`
+	PriceUnit                  *string                                      `json:"price_unit,omitempty"`
+	TestNumberFloat            *float32                                     `json:"test_number_float,omitempty"`
+	TestEnum                   *string                                      `json:"test_enum,omitempty"`
+	TestArrayOfIntegers        []int                                        `json:"test_array_of_integers,omitempty"`
+	TestArrayOfArrayOfIntegers [][]int                                      `json:"test_array_of_array_of_integers,omitempty"`
+	TestArrayOfObjects         *[]TestResponseObjectTestArrayOfObjectsInner `json:"test_array_of_objects,omitempty"`
```

Comment:
I don't think we want this naming change. But it might be okay if this is not something that impacts twilio-go or terraform-provider-twilio. Would need to generate those and check the diff.

Reply:
I think this was generated by [openapi-generator library](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/InlineModelResolver.java). Will check if I can change it back or if its impacts the generated file.

Reply:
update: some generated files in REST folder are affected by the additional "inner" keyword. E.g. PricingV1MessagingMessagingCountryInstanceInboundSmsPricesInner was generated

Reply:
Eek. Think we need to revert back this naming change.

Reply:
I don't think we can revert back the naming change unless we stick back to 5.4.0, as it's [hardcoded](modules/openapi-generator/src/main/java/org/openapitools/codegen/InlineModelResolver.java#L268) within their code. There's an [implementation](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/InlineModelResolver.java#L74) to set but it's still under their master branch.

Reply:
If it's just the filename that changes then we're fine.

Reply:
The struct name changes too `type VideoV1RoomRoomRecordingRuleRulesInner struct`